awx/main/migrations/0168_wfjt_prompt_all.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 3.2.13 on 2022-08-31 17:30
```

Comment:
I would prefer that the feature branches keep it limited to a single migration file. These are ultimately user-facing, but the reason for separating them is the notion of keeping AWX releasable, and if that really is the logic, then feature branches should not have any problem combining migration files.